public/scripts/createLabel.js

```diff
@@ -0,0 +1,36 @@
+async function postLabel(labelData) {
+    let response
+    try {
+        response = await fetch('/api/labels', {
+            method: 'post',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            credentials: 'include',
+            body: JSON.stringify(labelData)
+        })
+    } catch (error) {
+        console.log('Fetch Error :-S', error)
```

Comment:
Maybe you want to try a `console.error` instead, if you want the error the be displayed in the console?